src/txn/txn_recover.c

```diff
@@ -813,6 +813,9 @@ __recovery_file_scan_prefix(WT_RECOVERY *r, const char *prefix, const char *igno
 static int
 __recovery_file_scan(WT_RECOVERY *r)
 {
+    __wt_verbose_level_multi(r->session, WT_VERB_RECOVERY_ALL, WT_VERBOSE_INFO, "%s",
```

Comment:
There's a lot of emphasis on recovery with these changes, but the BF linked on the ticket indicated recovery wasn't the problem. What about adding some more in `wiredtiger_open`?

Reply:
I added a couple more logs in the flow of `wiredtiger_open`.

Reply:
Thanks - that's the kind of thing I had in mind when I made the ticket.